modules/navier_stokes/src/fvkernels/INSFVMomentumAdvection.C

```diff
@@ -133,7 +133,7 @@ INSFVMomentumAdvection::computeResidualsAndAData(const FaceInfo & fi)
                                            nullptr,
                                            &limiter_time};
     if (const auto [is_jump, eps_elem_face, eps_neighbor_face] =
-            NS::isPorosityJumpFace(epsilon(), fi, state);
+            NS::isPorosityJumpFace<ADReal>(epsilon(), fi, state);
```

Comment:
```suggestion
            NS::isPorosityJumpFace(epsilon(), fi, state);
```

Reply:
for these 3 calls the auto-detection is not working.

it gets hung up on the conversion epsilon, a FunctorBase, to a FunctorEnvelope (`Functor<T>`)

Reply:
Let's change `isPorosityJumpFace` to take `FunctorBase` then. Then we don't have an implicit conversion happening